packages/venia-ui/lib/components/WishlistPage/__tests__/wishlist.spec.js

```diff
@@ -90,6 +90,6 @@ test('hides visibility toggle', () => {
 });
 
 test('render no button when id is not set', () => {
-  const tree = createTestInstance(<Wishlist data={{}} />);
-  expect(tree.toJSON()).toMatchSnapshot();
-})
+    const tree = createTestInstance(<Wishlist data={{}} />);
```

Comment:
Test should match the new implementation.
```suggestion
    const tree = createTestInstance(<Wishlist />);
```